src/context/Global.tsx

```diff
@@ -260,14 +325,25 @@ const GlobalProvider = (props: { children: any }) => {
             );
         });
 
-    const setSwapStorage = (swap: SwapWithId) =>
+    migrateToChainSwaps(swapsForage)
```

Comment:
This would run and iterate over all swaps on every reload. How about introducing versioning?